include/dlaf/mc/triangular_solve_local.h

```diff
@@ -25,17 +25,20 @@ namespace dlaf {
 /// @brief Triangular Solve implementation on local memory, solving op(A) X = alpha B (when side == Left)
 /// or X op(A) = alpha B (when side == Right).
 ///
-/// @param side specifies whether op(A) appears on the \a Left or on the \a Right of dlaf::Matrix X
-/// @param uplo specifies whether the dlaf::Matrix A is a \a Lower or \a Upper triangular matrix
+/// @param side specifies whether op(A) appears on the \a Left or on the \a Right of matrix X
+/// @param uplo specifies whether the matrix A is a \a Lower or \a Upper triangular matrix
 /// @param op specifies the form of op(A) to be used in the matrix multiplication: \a NoTrans, \a Trans,
 /// \a ConjTrans
-/// @param diag specifies if the dlaf::Matrix A is assumed to be unit triangular (\a Unit) or not (\a NonUnit)
-/// @param alpha specifies the scalar alpha
-/// @tparam A refers to a triangular dlaf::Matrix object
-/// @tparam B refers to a dlaf::Matrix object
+/// @param diag specifies if the matrix A is assumed to be unit triangular (\a Unit) or not (\a NonUnit)
+/// @param a contains the triangular matrix A. Only the tiles of the matrix which contain the upper or
+/// the lower triangular part (depending on the value of uplo) are accessed in read-only mode (the
+/// elements are not modified).
+/// @param b on entry it contains the matrix B, on exit the matrix elements are overwritten with the
```

Comment:
You renamed the function parameters to `mat_a` and `mat_b`, therefore also the doc should be updated to use the same names...